client/App.vue

```diff
@@ -76,6 +93,12 @@ export default {
         search,
       });
     },
+    onNewsDismiss() {
+      localStorage.setItem(
+        'news-last-updated',
```

Comment:
Think 'news-last-updated' could be clearer since it's a per user value (not system level). Maybe 'news-last-viewed-at' and you just use Date.now() instead of date_modified of newsItem?

Reply:
I like 'news-last-viewed-at'. 

I was thinking about using new Date(), however if we were to put times on the dates published for articles and released throughout the day, it's possible to publish an article and not see it pop up. The advantage of taking the latest date on the feed is you won't skip an article.